packages/core/src/Constants/cfd-text.js

```diff
@@ -3,7 +3,7 @@ import { localize } from '@deriv/translations';
 export const CFD_TEXT = {
     dxtrade: () => localize('Deriv X'),
     mt5: () => localize('MT5'),
-    mt5_cfds_mfsa: () => localize('MT5 CFDs MFSA'),
+    mt5_cfds_mfsa: () => localize('MT5 CFDs'),
```

Comment:
can we also update the key to `mt5_cfds`?

Reply:
Done